frontend/src/components/input-output/configure-ds/ConfigureDs.jsx

```diff
@@ -151,7 +168,7 @@ function ConfigureDs({
   };
 
   const handleSubmit = () => {
-    if (!isTcSuccessful) {
+    if (isTcSuccessful) {
```

Comment:
@tahierhussain If test connection is successful, the current logic will throw an error.

Reply:
@Deepak-Kesavan Thanks for the catch. I had made this change for testing purpose. I have reverted the necessary changes.